lib/model/model.go

```diff
@@ -182,15 +183,11 @@ func (m *Model) StartFolder(folder string) {
 }
 
 func (m *Model) startFolderLocked(folder string) config.FolderType {
-	cfg, ok := m.folderCfgs[folder]
-	if !ok {
-		panic("cannot start nonexistent folder " + cfg.Description())
+	if err := m.checkFolderRunningLocked(folder); err == nil || err == errFolderMissing {
```

Comment:
`nil.Error()` will panic itself with a random thing, so you want something better.